src/assests/closeIcon.js

```diff
@@ -0,0 +1,7 @@
+const CloseIcon = props =>
+    <svg xmlns="http://www.w3.org/2000/svg" className="h-6 w-6" height='1.5rem' width='1.5rem'
```

Comment:
height, width, color to be passed from props, keep onClick on parent

Reply:
Refer this - [https://github.com/DemonDaddy22/Lorem-Gen-Extension/blob/master/src/assets/error.tsx](https://github.com/DemonDaddy22/Lorem-Gen-Extension/blob/master/src/assets/error.tsx)

Reply:
done!